Decred Wallet/en.lproj/Localizable.strings

```diff
@@ -410,3 +417,8 @@
 "history" = "history";
 "settings" = "Settings";
 "help" = "help";
+
+/* PIN Setup*/
+"next" = "Next";
+"create" = "Create";
+"back" = "Back";
```

Comment:
Where's this used? It's not in the `LocalizedStrings.swift` file.

Reply:
It is used directly on `Security.storyboard` on `RequestPinView`